content/blog/2021/04/20-jenkins-contributor-awards.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title: "Jenkins Contributor Awards - Nominations Open"
+tags:
+- awards
+- jenkins
+- cdcon
+author: cdfoundation
```

Comment:
Feel free to use your name as an author @zestyroxy :) Whatever works best

Reply:
I prefer the cd alias Mark created for us :)